rhv_checks.py

```diff
@@ -235,6 +236,63 @@ def check_storage_domain_attached_status(system, **kwargs):
         print("Ok: all Storage Domain(s) are Attached to Data Center(s): {}".format(okay))
         sys.exit(0)
 
+
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the hosts."""
+    okay, warning, critical, unknown, all = [], [], [], [], []
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
+        result = ssh_command(host.name, password, command)
+        host_data_list = []
+        host_data = {}
+        for data in result:
+            # if ' : ' in data:
+            if 'Host ID' in data or 'Engine status' in data or 'Hostname' in data:
```

Comment:
Can you please consider fetching Score as well as we discussed on the call?

Reply:
Update logic.